docs/getting_started/tutorial-hyperbolic-discounting.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>Loop with <code>zip</code> over mean max experience and labels to plot the three lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>